CONTRIBUTING.md

```diff
@@ -91,6 +91,45 @@ a new space — which isn't auto-deployed to — for testing and demoing specifi
 While we routinely demo our work on the development site, the FEC folks will
 conduct full reviews of the work slated for a release on our staging site.
 
+## Team's workflow
+### Issue and PR management
+1. Please pull an issue from the current sprint only (unless there is an
+   *URGENT* issue). If a non *URGENT* issue comes up, but you want to prioritize it, coordinate with the PMs or tech lead on Slack.
+2. Before starting,
```

Comment:
I'm not sure where [this link](https://github.com/fecgov/openFEC/blob/develop/CONTRIBUTING.md#development-issues-should-include) should go, but I think it might be broken

Reply:
Thanks @lbeaufort, I corrected the link. It's supposed to go to the #issues heading on this page, I saw that it outlined the issue checklist there.